src/renderers/webgl/WebGLMaterials.js

```diff
@@ -0,0 +1,678 @@
+import { BackSide } from "../../constants";
+
+/**
+ * @author taphos / https://github.com/taphos
```

Comment:
```js
/**
 * @author mrdoob / http://mrdoob.com/
 */
```
This is an excellent suggestion and PR, but if it were me, I would leave mrdoob as the only author. We don't typically give attribution for copy-paste refactoring.

Reply:
I am sure @taphos did not mean any disrespect, and was just "following the pattern".

Reply:
ok

Reply:
I see `@author` as being responsible for the file in the future, so if @taphos wants to opt-in... 😉